app/assets/stylesheets/sass/views/attendance/index.css.scss

```diff
@@ -0,0 +1,144 @@
+.sub-container {
+  height: 100%;
+  margin: 5% 5%;
+  width: 80%;
+ 
+  .search-row {
+    display: flex;
+    justify-content: space-between;
+     .month-option{
```

Comment:
Add a extra space between your class and your bracket, and add a extra space between `.search-row` and `.month-option`

Reply:
Thanks, Done!